courses/machine_learning/deepdive2/structured/labs/5a_train_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p>need to include</p><p>IMAGE=gcr.io/$PROJECT/babyweight_training_container</p><p>otherwise, not recognized from previous cell block</p><p> </p><p>(in solution notebook as well)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Thanks!</p>